.env

```diff
@@ -7,3 +7,6 @@ PROVE_BEGIN_BATCH=
 PROVE_END_BATCH=
 L2GETH_API_URL=
 ROLLUPSCAN_API_URL=
+GRPC_VERBOSITY=DEBUG
+GRPC_TRACE=all
+RUST_MIN_STACK=100000000
```

Comment:
Do we need all these?

Reply:
RUST_MIN_STACK is only needed. The rest will be removed.